task_processing/plugins/kubernetes/task_config.py

```diff
@@ -0,0 +1,52 @@
+import uuid
+from typing import Sequence
+from typing import TYPE_CHECKING
+
+from pyrsistent import field
+from pyrsistent import m
+from pyrsistent import PMap
+from pyrsistent import pmap
+from pyrsistent import PVector
+from pyrsistent import pvector
+from pyrsistent import v
+
+from task_processing.interfaces.task_executor import DefaultTaskConfigInterface
+
+VOLUME_KEYS = set(['mode', 'container_path', 'host_path'])
+
+def valid_volumes(volumes):
+    for vol in volumes:
+        if set(vol.keys()) != VOLUME_KEYS:
+            return (
+                False,
+                'Invalid volume format, must only contain following keys: '
+                '{}, was: {}'.format(VOLUME_KEYS, vol.keys())
+            )
+    return (True, None)
+
+class KubernetesTaskConfig(DefaultTaskConfigInterface):
+    uuid = field(type=(str, uuid.UUID), initial=uuid.uuid4)
```

Comment:
we can probably omit this and just use the inherited uuid from the superclass if we ever need a uuid, right?

Reply:
Omitted.